public/app.js

```diff
@@ -6,71 +6,106 @@ let width = window.innerWidth;
 let height = window.innerHeight;
 let factor;
 let responsiveSize;
+let kittyPool = [];
+let kittyPoolLimit = 10;
+
+/**
+ * Preloads some cat images to be used within the game
+ */
+(function preloadCats() {
+  kittyPool.push((new Image().src = 'cat.png'));
+
+  function getRandomSize() {
```

Comment:
Random size results in some images appearing stretched, can you resolve this issue?